### PR TITLE
Update request to 2.72.x

### PR DESCRIPTION
**Pull request by ziluvatar**
- Update fork from root repository
- Upgrade request dependency due to a security issue in one of its dependencies (hawk: https://nodesecurity.io/advisories/77).

There is already an [open PR in twilio repository to fix it](https://github.com/twilio/twilio-node/pull/167), meantime we can update our fork.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "url": "https://github.com/twilio/twilio-node.git"
   },
   "dependencies": {
-    "request": "2.55.x",
+    "request": "2.70.x",
```
Comment (dafortune) [inline package.json]: Minor detail, but I think you can update to `^2.72.x` as https://github.com/twilio/twilio-node/pull/167/files

**Comment by dafortune** [comment]
😍 if this end ups merged I'd for sure use it in Guardian. All the commits except last one (yours) are the ones in the original repo, right?

File: package.json (hunk 1 above)
Comment (ziluvatar) [inline package.json]: Haha true, I was reviewing the history of some package.json and I mixed up the final version to set.

**Comment by ziluvatar** [comment]
Yes, I only added the last commit, the rest are from the original repo. This merge is for auth0 fork, I saw you had your own fork but I realized later there were a common one.

**Comment by hernanhht** [comment]
LGTM

**Comment by ziluvatar** [comment]
We will use @dafortune fork for now (dafortune/twilio-node#update-request-library). In the future we will "re-create" this fork using last version from twilio.

**Pull request closed without merging**